x-pack/plugins/profiling/public/app.tsx

```diff
@@ -68,18 +71,9 @@ const commonlyUsedRanges: CommonlyUsedRange[] = [
   },
 ];
 
-interface TimeRange {
-  start: string;
-  end: string;
-  isoStart: string;
-  isoEnd: string;
-  unixStart: number;
-  unixEnd: number;
-}
-
 function buildTimeRange(start: string, end: string): TimeRange {
-  const timeStart = dateMath.parse(start);
-  const timeEnd = dateMath.parse(end);
+  const timeStart = dateMath.parse(start)!;
+  const timeEnd = dateMath.parse(end)!;
```

Comment:
`!` tells TypeScript the value is non-nullish. It's somewhat risky, because we don't know if it's actually non-nullish when this code is evaluated, but that previously was the case too, and the error was not handled either, so I've opted for simplicity here.